doc-content/enterprise-only/openshift/offline-settings-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='offline-settings-proc']
+= Mounting the custom settings file to access your Maven mirror repository
+
+If your OpenShift environment does not have a connection to the public Internet, you must mount the custom settings file that you prepared according to <<offline-repo-proc>> in your {KIE_SERVER} deployment.
+
+ifeval::["{context}"=="openshift-immutable"]
+This procedure does not apply to source to image (S2I) {KIE_SERVER} deployment.
+endif::[]
+
+. In a command line window, change to the directory where the custom `settings.xml` file is located.
+
+. Create an OpenShift configuration map from the file. Run the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap maven_settings --from-file=settings.xml=settings.xml
+----
++
+ifeval::["{context}"!="openshift-managed"]
+. Mount the configuration map on the {KIE_SERVER} deployment configuration:
++
+[subs="attributes,verbatim,macros"]
+----
+ oc set volume dc/_myapp_-kieserver --add --type configmap --configmap-name maven_settings --mount-path=.m2 --name=maven_settings
```

Comment:
Please add a namespace reference to this command - `-n NAMESPACE`.

Reply:
There is also a need to specify subpath to isolate config map just to one file, not whole folder.
The command which worked for me is this:
`oc set volume dc/myapp-kieserver --add --type configmap --configmap-name maven-settings --mount-path=/home/jboss/.m2/settings.xml --sub-path=settings.xml --name=maven-settings -n NAMESPACE`